app.js

```diff
@@ -115,9 +115,19 @@ app.get('/inventory', function(req, res) {
         }
 
         console.log(results)
-        // Check for items that are low on the shelf and set .shelf_low to true if they are
+
+        // Check for not active item
+        results.forEach(function(value, index) {
+            if (value.active != 0) {
+                value.not_catalog = false
+            } else {
+                value.not_catalog = true
+            }
+        })
+
+        // Check for "Active" items that are low on the shelf and set .shelf_low to true if they are
         results.forEach(function(value, index) {
-            if (value.shelf_quantity < value.shelf_min_threshold) {
+            if (value.shelf_quantity < value.shelf_min_threshold && value.active != 0) {
```

Comment:
```suggestion
            if (value.shelf_quantity < value.shelf_min_threshold && value.active == 1) {
```

Reply:
Why change to value.active == 1 ? For boolean, you can insert number besides 0 and 1. All non-zero value is true.  If someone insert 2 instead of 1 for active attribute, value.active == 1 will not work.

Reply:
Ah, good point, I forgot. I was thinking more on the lines of "is true" is more straight forward than "is not false". But you're right, if we want to consider the possibility of other truthy values for this attribute, best to leave how you worded it.